dotnet-web/pipeline.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+  - label: ":package: Restore dependencies"
+    command: "dotnet restore"
+    plugins:
+      - docker#v5.9.0:
+          image: "mono:latest"
```

Comment:
We should reference this `mono` docker image somewhere in the README. It wasn't immediately obvious to me what it was.

Reply:
Newbie mistake from my end 😅 there's no need for this so I've removed it.